platform-dom/src/test/java/com/softicar/platform/dom/elements/AbstractDomValueSelectTest.java

```diff
@@ -126,7 +126,7 @@ private void assertDisplayNames(List<Pair<Integer, IDisplayString>> values, Stri
 			if (displayNames[i] != null) {
 				IDisplayString expected = IDisplayString.create(displayNames[i]);
 				IDisplayString actual = values.get(i).getSecond();
-				assertTrue(expected.compareTo(actual) == 0);
+				assertTrue(expected.toString().compareTo(actual.toString()) == 0);
```

Comment:
```suggestion
				assertEquals(expected.toString(), actual.toString());
```
maybe add an `assertEquals(IDisplayString,IDisplayString)` to `Asserts`?

Reply:
added that